regression-test/suites/node_p0/test_backend.groovy

```diff
@@ -39,4 +39,14 @@ suite("test_backend") {
         result = sql """SHOW BACKENDS;"""
         logger.info("result:${result}")
     }
+
+    if (context.config.jdbcUser.equals("root")) {
+        def result = sql """SHOW BACKENDS;"""
+        logger.info("show backends result:${result}")
+        def beId1 = result[0][0]
+        result = sql """ALTER SYSTEM DECOMMISSION BACKEND "${beId1}" """
```

Comment:
after alter/cancel,  check be decommission status from  'show backends''s .

Reply:
fixed